mautic/mautic
# Try to find out the existing lead with device fingerprint.

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | No
| New feature? | Yes
| Related user documentation PR URL | n/a
| Related developer documentation PR URL | n/a
| Issues addressed (#s or URLs) | n/a
| BC breaks? | n/a
| Deprecations? | No

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Trying to leverage the device fingerprint to identify the lead when it fails to find out the lead by session.

#### Steps to test this PR:
1. Enable the fingerprint tracking option from Configuration -> Landing Page Settings -> 3rd party website tracking code -> Identify returning visitor by device fingerprint
2. Open a 3rd party page which has Mautic tracking code embedded, so that to generate a page git record.
3. Clear the session data from the browser.
4. Repeat Step 1, notice that Mautic should be able to recognize that the second page hit is from the same lead.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
n/a

#### List deprecations along with the new alternative:
n/a

#### List backwards compatibility breaks:
n/a
